docs/Using-Fleet/Fleet-desktop.md

```diff
@@ -0,0 +1,24 @@
+# Fleet Desktop
+- [Installing Fleet Desktop](#installing-fleet-desktop)
+- [Upgrading Fleet Desktop](#upgrading-fleet-desktop)
+- [Custom Transparency Link](#custom-transparency-link)
+
+Fleet Desktop is a menubar icon available on macOS, Windows, and Linux. 
+
+At its core, Fleet Desktop gives your end users visibility into the security posture of their machine. This unlocks two key benefits:
+* Self-remediation: end users can see which policies they are failing and resolution steps, reducing the need for IT and security teams to intervene
+* Scope Transparency: end users can see what the Fleet agent can do on their machines, eliminating ambiguity between end users and their IT and security teams
+
+## Installing Fleet Desktop
+For information on how to install Fleet Desktop, visit: [Adding Hosts](https://fleetdm.com/docs/using-fleet/adding-hosts#fleet-desktop).
+
+## Upgrading Fleet Desktop
+Once installed, Fleet Desktop will be automatically updated via Orbit. To learn more, visit: [Self-managed agent updates](https://fleetdm.com/docs/deploying/fleetctl-agent-updates#self-managed-agent-updates).
+
+## Custom transparency link
+For organizations with complex security postures, they can direct end users to a resource of their choice to serve custom content.
+
+> The custom transparency link is only available for users with Fleet Premium
+
+To turn on the custom transparency link, in the Fleet GUI, click on your profile in the top right, select "Settings."
+In the settings page, under "Organization Settings" select "Fleet Desktop." Use the "Custom transparency URL" text input to specify the custom URL.
```

Comment:
@zhumo build is failing because you need to add a `<meta name="pageOrderInSection" value="REPLACE_ME">` in this file.

The `value` attribute determines the order in which this item will appear in lists like this one: https://fleetdm.com/docs

Reply:
Thank you!